docs/jwst/fringe/main.rst

```diff
@@ -6,9 +6,10 @@ Description
 
 The ``fringe`` step applies a fringe correction to MIRI MRS images.
 In particular, the SCI array from a fringe reference file is divided into the
-SCI and ERR arrays of the science data set. Only pixels that have valid (non-NaN)
+SCI and ERR arrays (and likewise squared for variance arrays) of the science data set.
```

Comment:
I'd suggest making this its own sentence for clarity, e.g.
```suggestion
SCI and ERR arrays of the science data set. The variance arrays are divided by the square of the reference data.
```

Reply:
Re: code style updates -- Just some that really bothered me as I go but I also don't want this to be a full-fledge ruff PR. So if the tiny updates bother you, I can revert them.

Reply:
They don't bother me, I just wanted to make sure I shouldn't point out additional things to change :)